reactph/reactjsph-website
# Use Fb Sdk in the client side

**Description**
- strategize a way to use fb sdk in the client side instead at build time.

**Summary of Changes**
- create key hooks such as `useFacebookSdk` and `useFacebookPageEventQuery`

`useFacebookSdk`
- loads the facebook sdk through javascript requests rather than html inclusion to make sure we can tract the loading / error of this module so that there's no react lifecycle problems.

`useFacebookPageEventQuery`
- provides necessary states to keep track of the fb api call.

- created a fb token boundary component to notify devs in order to render the meetup section component. will fail gracefully and not render the component if such env variable does not exist.

- updated `meetupSection.js`

**How to Test**

Test cases
- Run normally without exporting anything. Result should be the meetup section is not rendered as well as key hints at the consoles on what to do.
- Run normally exporting the `FACEBOOK_GRAPH_TOKEN`. Result should show the meetup section.

Closes #96
